### PR TITLE
Wiki

### PR DESCRIPTION
**Pull request by mcdfinn**
Added Heath Potion wiki page, as well as fixing Turret's page.

### CODE REVIEW

**Review by TheCoub — APPROVED** [review]

File: wiki/heath_potion.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Turrets
```
Comment (jyan-zarate) [inline wiki/heath_potion.md]: Are you high? This file is about Health Potions so the title should be `Health Potions`

File: wiki/heath_potion.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Turrets 
+=======
+
+* Icon: `♥`
+* Buy with: `l, 1, *`
+* Use with: `u, 1, *`
+* Cost: `50 Ore`
```
Comment (jyan-zarate) [inline wiki/heath_potion.md]: Use this following formatting instead. The below values are placeholders.

* Cost:
    * Construction Cost: 1
    * Buy from Pharmacy with `M` standing: 5
    * Buy from Pharmacy with `A` standing: 10
    * Buy from Pharmacy with `N` standing: 15
    * Buy from Pharmacy with `E` standing: 20

File: wiki/heath_potion.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Services Offered
+
+Health Potions give the user an instant boost of 15 health directly to their health bar.
```
Comment (jyan-zarate) [inline wiki/heath_potion.md:16]: To their health bar? That kinda sounds weird. Say this instead

`Health Potions give their user an instant health boost of 15.`

File: wiki/heath_potion.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Owner Information
+
+Health Potions are acquired from the Pharmacy,
+a corporation owned building. They can be bought by anyone, with varying margins of profit. Members of a corporation (M),(@) gain you no profit. Allies (A) net you 1 ore, Neutrals (N) 5 ore, and Enemies (E) get you 10 ore per potion.
+#### Origin
```
Comment (jyan-zarate) [inline wiki/heath_potion.md:23]: Add a new line before `### Origin`

File: wiki/heath_potion.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+* Use with: `u, 1, *`
+* Cost: `50 Ore`
+
+#### Services Offered
+
+Health Potions give the user an instant boost of 15 health directly to their health bar.
+
+#### Owner Information
+
+Health Potions are acquired from the Pharmacy,
```
Comment (jyan-zarate) [inline wiki/heath_potion.md:20]: Why a new line for the comma?

File: wiki/heath_potion.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+a corporation owned building. They can be bought by anyone, with varying margins of profit. Members of a corporation (M),(@) gain you no profit. Allies (A) net you 1 ore, Neutrals (N) 5 ore, and Enemies (E) get you 10 ore per potion.
```
Comment (jyan-zarate) [inline wiki/heath_potion.md:21]: For new sentences, just use a new line as it'll render the same but look neater when editing. Example:

`Line 1 - The bobcat is very silly.`
`Line 2 - The children loved to play with the bobcat, until it got fed up with their unclean smiles and mauled a child.`

renders into

`The bobcat is very silly. The children loved to play with the bobcat, until it got fed up with their unclean smiles and mauled a child.`

**Review by JCharante — CHANGES_REQUESTED** [review]
Let me know when you address these comments.

**Review by JCharante — APPROVED** [review]

**Pull request merged**